save-common/src/commonMain/kotlin/com/saveourtool/save/core/config/FixFormats.kt

```diff
@@ -0,0 +1,16 @@
+/**
+ * Classes that describe format of expected and actual fixes
+ */
+
+@file:Suppress("FILE_NAME_MATCH_CLASS", "MatchingDeclarationName")
+
+package com.saveourtool.save.core.config
+
+/**
+ * Possible formats of actual set of fixes, provided by user
+ */
+enum class ActualFixFormat {
```

Comment:
is this flag used in fix and warn?

Reply:
`FixAndWarn` plugin actually have no own configuration, it use `fix` and `warn` configs as subsections, so there is no need of this flag in `FixAndWarn` directly. If it used in `fix` section, then it will be used in `FixAndWarn`